framework/core/js/src/forum/components/PostMeta.js

```diff
@@ -55,6 +55,8 @@ export default class PostMeta extends Component {
    * @returns {string}
    */
   getPermalink(post) {
-    return app.forum.attribute('baseUrl') + app.route.post(post);
+    // URL.hostname transformation is necessary to avoid duplicating
+    // path in subdirectory installs.
+    return URL(app.forum.attribute('baseUrl')).hostname + app.route.post(post);
```

Comment:
This will not keep URLs which contain ports, nor the protocol. We should use `.origin` instead.


```suggestion
    return new URL(app.forum.attribute('baseUrl')).origin + app.route.post(post);
```